src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -60,19 +59,19 @@
         </p>
         <DatePicker
           v-if="hasCalendar && showCalendar"
-          v-model="selectedCalendarRange"
+          v-model="calendarVModel"
           :drag-attribute="calendarDragAttributes"
           is-expanded
-          :is-range="range"
+          :is-range="!isSingleDatepicker ? true : false"
```

Comment:
suggestion: I think this can be simplified?
```suggestion
          :is-range="!isSingleDatepicker"
```